targets/targets.json

```diff
@@ -2363,12 +2363,32 @@
             "lpticker_delay_ticks": 0
         },
         "device_has_add": [
-            "ANALOGOUT",
             "SERIAL_ASYNCH",
             "FLASH",
             "MPU"
         ]
     },
+    "MCU_STM32G031xx": {
+        "inherits": [
+            "MCU_STM32G0"
+        ],
+        "public": false,
+        "extra_labels_add": [
```

Comment:
Maybe you could add:
"supported_application_profiles": [
            "bare-metal"
        ],
To make explicit that only bare metal is supported ?

Reply:
Added to NUCLEO_G031K8 target itself. See 167241a